src/Compiler/Driver/CompilerImports.fs

```diff
@@ -1172,7 +1172,14 @@ and [<Sealed>] TcImports
         match (Some ccu.Contents, nsname) ||> List.fold matchNameSpace with
         | Some ns ->
             match Map.tryFind tname ns.ModuleOrNamespaceType.TypesByMangledName with
-            | Some _ -> true
+            | Some e ->
+                if publicOnly then
```

Comment:
Should this be `AccessibleFromEverywhere`, `AccessibleFromSomeFSharpCode` or `AccessibleFromSomewhere`? I.e. do we care about internals + `InternalsVisibleTo`? Probably not for `IsReadOnlyAttribute` in particular, but maybe for others?